paper/main.tex

```diff
@@ -594,8 +595,8 @@ \section*{Results}
     the relative weights of each word (via their relative sizes) reflected by the blend of topics
     represented at the locations of the stars on the maps. The words' colors
     indicate how much each word is weighted, on average, across all timepoints'
-    topic vectors in the \textit{Four Fundamental Forces} (blue) and
-    \textit{Birth of Stars} (green) videos, respectively.}
+    topic vectors in the \textit{Four Fundamental Forces} (bluer coloring) and
```

Comment:
outstanding change <span>#</span>2:
"blue" --> "bluer"; "green" --> "greener"

Could take or leave this one